configs/README.md

```diff
@@ -1,15 +1,23 @@
 ## Configuration File
-Example configuration files are provided in this directory. To build and run the given examples see the instructions [here](https://github.com/NOAA-OWP/cfe/blob/master/INSTALL.md). A detailed description of the parameters for model configuration (i.e., initialize/setup) is provided below. The asterisk (*) denotes calibratable parameters.
+Example configuration files are provided in this directory.
+To build and run the given examples see the instructions [here](https://github.com/NOAA-OWP/cfe/blob/master/INSTALL.md).
+A detailed description of the parameters for model configuration (i.e., initialize/setup) is provided below.
+> [!NOTE]
+> The asterisk (*) denotes calibratable parameters.
+> Note, calibratable parameters are passed via BMI using the _name next to the asterisk_.
+>
+> Slashes (/) in the variable column denote that a parameter can be specified using either name.
+> For example, `soil_params.b` can equivalently be provided as `b`.
 
 | Variable | Datatype |  Limits  | Units | Role | Process | Description |
 | -------- | -------- | ------ | ----- | ---- | ------- | ----------- |
 | forcing_file | *char* | 256  |   | filename |   | path to forcing inputs csv; set to `BMI` if passed via `bmi.set_value*()`  |
 | soil_params.depth | *double* |   | meters [m]| state |  | soil depth  |
```

Comment:
for consistency, we should follow the same style for `soil_params.depth` as well

Reply:
In an ideal world, I would have added short forms of all of the `soil_params`, but I couldn't for `soil_params.expon` because `expon` already exists... so, I only added short forms for calibratable parameters. @ajkhattak, thoughts on how to get around this?

Reply:
okay, I see what you mean... Let's discuss it with other folks. I would suggest something like

soil_param_name (e.g., `soil_storage`)
gw_param_name (e.g., `gw_storage`)

since the parameter `expon` is used in both soil and groundwater reservoirs, so following the above, we use
`soil_expon` and `gw_expon`.

we can update the names for calibratable parameters too, so they are consistent with their names in the config file
.

@SnowHydrology @hellkite500